packages/ui/src/components/OpenApi/OpenApiForm.tsx

```diff
@@ -108,10 +116,13 @@ export const OpenApiForm: FunctionComponent = () => {
 
     const openApi: OpenApi = JSON.parse(spec);
 
-    for (const path in openApi.paths) {
-      for (const method in openApi.paths[path]) {
-        const operationId = openApi.paths[path][method]['operationId'];
```

Comment:
While this worked, the OpenApi spec defines other fields that we might be iterating over if present:
```typescript
export interface OpenApiPath {
    /**
     * Allows for an external definition of this path item. The referenced structure MUST be in the format of a Path Item Object.
     * If there are conflicts between the referenced definition and this Path Item's definition, the behavior is undefined.
     */
    $ref?: string;
    /** An optional, string summary, intended to apply to all operations in this path. */
    summary?: string;
    /** An optional, string description, intended to apply to all operations in this path. CommonMark syntax MAY be used for rich text representation. */
    description?: string;
    /** A definition of a GET operation on this path. */
    get?: OpenApiOperation;
    /** A definition of a PUT operation on this path. */
    put?: OpenApiOperation;
    /** A definition of a POST operation on this path. */
    post?: OpenApiOperation;
    /** A definition of a DELETE operation on this path. */
    delete?: OpenApiOperation;
    /** A definition of a OPTIONS operation on this path. */
    options?: OpenApiOperation;
    /** A definition of a HEAD operation on this path. */
    head?: OpenApiOperation;
    /** A definition of a PATCH operation on this path. */
    patch?: OpenApiOperation;
    /** A definition of a TRACE operation on this path. */
    trace?: OpenApiOperation;
    /** An alternative server array to service all operations in this path. */
    servers?: OpenApiServer[];
   ...
}
```

So better to iterate only on the intended methods, hence the need for the valid methods array above.